azkaban-common/src/test/java/azkaban/utils/TestUtils.java

```diff
@@ -35,18 +35,14 @@
  */
 public class TestUtils {
 
-  public static File getFlowDir(final String projectName, final String flow) {
-    return ExecutionsTestUtil.getFlowFile(projectName, flow + ".flow");
-  }
-
   public static User getTestUser() {
     return new User("testUser");
   }
 
   /* Helper method to create an ExecutableFlow from serialized description */
   public static ExecutableFlow createExecutableFlow(final String projectName,
       final String flowName) throws IOException {
-    final File jsonFlowFile = getFlowDir(projectName, flowName);
+    final File jsonFlowFile = ExecutionsTestUtil.getFlowFile(projectName, flowName + ".flow");
```

Comment:
Should ExecutionsTestUtil and TestUtils have methods similar? should they be merged in some way?

Reply:
ExecutionsTestUtil is specific for execution related tests. 
Any benefit in combining them?
I prefer smaller classes.